.agents/skills/ss14-documentation-writing/SKILL.md

```
@@ -0,0 +1,13 @@
+name: ss14-documentation-writing
+description: A practical documentation standard in Space Station 14 for C#, SWSL, YAML and FTL: how to write `<summary>`, when to add explanatory comments, how to document partial systems, and how to avoid noisy documentation. Use it when writing, refactoring and reviewing documentation in code, prototypes and localization.
+metadata:
+  source_skill: "../../../.agent/skills/ss14-documentation-writing/SKILL.md"
+---
+
+# Codex Bridge
+
+Canonical skill file:
+../../../.agent/skills/ss14-documentation-writing/SKILL.md.
+
+Load references/scripts/assets from the same source skill directory.
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)